admin_guide/image_signatures.adoc

```diff
@@ -252,23 +252,35 @@ link:https://github.com/containers/image/blob/master/docs/atomic-signature.md#th
 === Importing Image Signatures Automatically from Signature Stores
 
 {product-title} can automatically import image signatures if a signature
-store is configured on all {product-title} master nodes through the _registries
-configuration directory_ located in the *_/etc/containers/registries.d_*. 
+store is configured on all {product-title} master nodes through the registries
+configuration directory. 
 
-The default registries configuration directory is the *_/etc/containers/registries.d/default.yaml_* file. 
+The registries configuration directory contains configuration 
+for various registries (servers storing remote container images), 
+and for content stored in them. The single directory ensures that the configuration 
+does not have to be provided in command-line options for each command, 
+and so that it can be shared by all users of containers/image.
+
+The default registries configuration directory is
+located in the *_/etc/containers/registries.d/default.yaml_* file. 
 
 A sample configuration that will cause image signatures to be imported
 automatically for all Red Hat images:
 
 ----
 docker:
   registry.access.redhat.com:
-    sigstore: https://access.redhat.com/webassets/docker/content/sigstore
+    sigstore: https://access.redhat.com/webassets/docker/content/sigstore <1>
 ----
 
-Note that all signatures imported automatically by {{product-title}} will be
+<1> Defines the URL of an signature store. This URL is used for reading existing signatures.
+
+[NOTE] 
+====
+Signatures imported automatically by {product-title} will be
 _unverified_ by default and will have to be verified by image administrators.
+====
 
 For more details about the registries configuration directory, see
 link:https://github.com/containers/image/blob/master/docs/registries.d.md[Registries Configuration Directory] 
-in the *containers/image* library documentation.
+in *containers/image* library documentation.
```

Comment:
this should be 

"see the containers/image library"

Reply:
Maybe I should remove the whole phrase and go with _see
link:https://github.com/containers/image/blob/master/docs/registries.d.md[Registries Configuration Directory]._ ??